README.md

```diff
@@ -204,3 +204,13 @@ Each label name is generated with [Lodash template](https://lodash.com/docs#temp
 The `releasedLabels` ```['released<%= nextRelease.channel ? ` on @\${nextRelease.channel}` : "" %> from <%= branch.name %>']``` will generate the label:
 
 > released on @next from branch next
+
+#### addReleases
+
+When set this is option will append all additional releases except the github release to the body of the github release.
+
+Valid values for this option are `false`, `top` or `bottom` where the releases will be added on the top when set to `top` and on the bottom when set to `bottom`.
```

Comment:
I think `top` and `bottom` are self-explanatory enough? I would add quotes to make clear that `false` should not be a string.

```suggestion
Valid values for this option are `false`, `"top"` or `"bottom"`.
```